apps/aehttp/priv/swagger.json

```diff
@@ -151,6 +151,37 @@
         },
         "security" : [ ]
       }
+    },
+    "/account/balance" : {
+      "get" : {
+        "tags" : [ "external" ],
+        "description" : "Get account's balance",
+        "operationId" : "GetAccountBalance",
+        "consumes" : [ "application/json" ],
+        "produces" : [ "application/json" ],
+        "parameters" : [ {
+          "name" : "pub_key",
+          "in" : "query",
+          "description" : "Public key to extract balance for",
+          "required" : true,
```

Comment:
This means that client needs to know pubkey before enquiring. How does client know key?
Alternative would be to make this parameter optional, and if not specified simply call `aec_keys:pubkey()`

Reply:
@lucafavatella I am merging this, @velzevur please add /getbalance going to current account